posthog/api/test/test_instance_settings.py

```diff
@@ -86,6 +88,22 @@ def test_update_integer_setting(self):
         self.assertEqual(response.json()["value"], 48343943943)
         self.assertEqual(getattr(config, "ASYNC_MIGRATIONS_ROLLBACK_TIMEOUT"), 48343943943)
 
+    @patch("ee.clickhouse.models.session_recording_event.sync_execute")
```

Comment:
Let's not do this. instead:

```
@snapshot_clickhouse_queries
```

and check the ttl using a `system.tables` query :)